kubeflow/trainer/backends/container/adapters/docker_test.py

```
@@ -0,0 +1,619 @@
+
+    if test_case.expected_error:
+        with patch.dict(sys.modules, {"docker": None}), pytest.raises(ImportError):
+            from importlib import reload
+
+            import kubeflow.trainer.backends.container.adapters.docker as mod
+
+            reload(mod)
+            mod.DockerClientAdapter()
+    else:
+        host = test_case.config["host"]
+        adapter, mock_docker = _create_adapter(host=host)
```

Ruff (0.15.20)

[warning] Unpacked variable `adapter` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)